stages/org.osbuild.ostree.post-copy

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/python3
+"""Apply post-copy updates to an ostree repo/deployment
+
+The way osbuild works the ostree deployment is built in a chroot and
+stored as a regular directory of files before finally being copied to
+the physical filesystem. This means that for example, ostree fs-verity
+support doesn't work, as the fs-verity setting of files is not copied.
+
+To support fs-verity in generated images you have to run this stage
+after copying the final ostree tree onto the target filesystem.
+
+Note: Ensure the target filesystem supports fs-verity. See e.g.  the
+`verity` option in org.ostbuild.mkfs.ext4.
+
+"""
+
+import os
+import sys
+
+import osbuild.api
+from osbuild.util import ostree
+
+SCHEMA_2 = r"""
+"options": {
+  "additionalProperties": false,
+  "properties": {
+  }
+},
+"devices": {
+  "type": "object",
+  "additionalProperties": true
+}
+"mounts": {
+  "type": "array"
+},
+"""
+
+
+def main(args, options):
+    mounts = args["mounts"]
+    root = mounts.get("root", {}).get("path")
+
+    ostree.cli("admin", "post-copy", sysroot=root)
```

Comment:
It might make sense to have `sysroot` as a stage option since the names of the mounts don't really have any special meaning.  We might, for example, use mount names that are just the mountpoint (or an escaped version of it).

Reply:
Would that just specify a mount name, or a path too?

Reply:
just the path that you need for `ostree.cli("admin", "post-copy", sysroot=...)`